docs/cart-and-checkout-blocks/available-filters/available-filters.md

```diff
@@ -17,64 +17,64 @@ tags: reference
--   [`subtotalPriceFormat`](./available-filters/cart-line-items.md#subtotalpriceformat)
+-   [`cartItemClass`](./cart-line-items.md#cartitemclass)
+-   [`cartItemPrice`](./cart-line-items.md#cartitemprice)
+-   [`itemName`](./cart-line-items.md#itemname)
+-   [`saleBadgePriceFormat`](./cart-line-items.md#salebadgepriceformat)
+-   [`showRemoveItemLink`](./cart-line-items.md#showremoveitemlink)
+-   [`subtotalPriceFormat`](./cart-line-items.md#subtotalpriceformat)
 
 The following screenshot shows which parts the individual filters affect:
 
 ![Cart Line Items](https://woocommerce.com/wp-content/uploads/2023/10/Screenshot-2023-10-26-at-13.12.33.png)
 
 ## Order Summary Items filters
 
-The following [Order Summary Items filters](./available-filters/order-summary-items.md) are available:
+The following [Order Summary Items filters](./order-summary-items.md) are available:
 
--   [`cartItemClass`](./available-filters/order-summary-items.md#cartitemclass)
--   [`cartItemPrice`](./available-filters/order-summary-items.md#cartitemprice)
--   [`itemName`](./available-filters/order-summary-items.md#itemname)
--   [`subtotalPriceFormat`](./available-filters/order-summary-items.md#subtotalpriceformat)
+-   [`cartItemClass`](./order-summary-items.md#cartitemclass)
+-   [`cartItemPrice`](./order-summary-items.md#cartitemprice)
+-   [`itemName`](./order-summary-items.md#itemname)
+-   [`subtotalPriceFormat`](./order-summary-items.md#subtotalpriceformat)
 
 The following screenshot shows which parts the individual filters affect:
 
 ![Order Summary Items](https://woocommerce.com/wp-content/uploads/2023/10/Screenshot-2023-10-26-at-16.29.45.png)
 
 ## Totals Footer Item filter
 
-The following [Totals Footer Item filter](./available-filters/totals-footer-item.md) is available:
+The following [Totals Footer Item filter](./totals-footer-item.md) is available:
 
--   [`totalLabel`](./available-filters/totals-footer-item.md#totallabel)
--   [`totalValue`](./available-filters/totals-footer-item.md#totalvalue)
+-   [`totalLabel`](./totals-footer-item.md#totallabel)
+-   [`totalValue`](./totals-footer-item.md#totalvalue)
 
 ## Checkout and place order button filters
 
-The following [Checkout and place order button filters](./available-filters/checkout-and-place-order-button.md) are available:
+The following [Checkout and place order button filters](./checkout-and-place-order-button.md) are available:
 
--   [`proceedToCheckoutButtonLabel`](./available-filters/checkout-and-place-order-button.md#proceedtocheckoutbuttonlabel)
--   [`proceedToCheckoutButtonLink`](./available-filters/checkout-and-place-order-button.md#proceedtocheckoutbuttonlink)
--   [`placeOrderButtonLabel`](./available-filters/checkout-and-place-order-button.md#placeorderbuttonlabel)
+-   [`proceedToCheckoutButtonLabel`](./checkout-and-place-order-button.md#proceedtocheckoutbuttonlabel)
+-   [`proceedToCheckoutButtonLink`](./checkout-and-place-order-button.md#proceedtocheckoutbuttonlink)
```

Comment:
Verify this type of link works - unsure if the parser can translate markdown -> docs site links that have hash property.

Reply:
I checked other docs had them written out like this.